frontend/tests/controllers/CourseCloneTest.php

```diff
@@ -58,6 +58,13 @@ public function testCreateCourseClone() {
         ]));
 
         $this->assertEquals($courseAlias, $courseClonedData['alias']);
+        $courseCloneLog = \OmegaUp\DAO\CourseCloneLog::getAll();
+        $coursesLog = array_filter(
+            $courseCloneLog,
+            fn (\OmegaUp\DAO\VO\CourseCloneLog $courseLog) =>
+                $courseLog->course_id === $courseData['course_id']
+        );
+        $this->assertCount(1, $coursesLog);
```

Comment:
aquí podemos usar [`assertArrayContainsWithPredicateExactlyOnce()`](https://github.com/omegaup/omegaup/blob/2f3ba4de5cb1a9f907f193fd084d34bdc5930e2b/frontend/tests/ControllerTestCase.php#L285):

```suggestion
        $this->assertArrayContainsWithPredicateExactlyOnce(
            \OmegaUp\DAO\CourseCloneLog::getAll(),
            fn (\OmegaUp\DAO\VO\CourseCloneLog $courseLog) =>
                $courseLog->course_id === $courseData['course_id']
        );
```

Reply:
Listo